### PR TITLE
[GEN-474, GEN-500, GEN-721, GEN-379] Include data guide generation in nextflow scripts

### PR DESCRIPTION
**Pull request by thomasyu888**
- Add in new quarto tool that generates the data guide.
- Add create data guide to nextflow step
- This was tested on both consortium and public within nextflow tower

Next step is to deprecate data guide generation from GENIE python package

### CODE REVIEW

File: scripts/data_guide/data_guide.qmd [hunk 1]
```diff
@@ -0,0 +1,620 @@
...
+  * variant_classifications: List of types of variants that are reported for this assay.
+  * gene_padding: Number of base pairs to add to exon endpoints for the inBED filter.
+  * alteration_types: List of alteration types. specimen_type: List of specimen types.
+  * specimen_tumor_cellularity: Tumor Cellularity Cutoff
+  * calling_strategy: Tumor only or tumor normal.
+  * coverage: List of coverage
+
+data_cna_hg19.seg
+
+* Description: Segmented copy number data. Note: Not all centers contributed copy number data to GENIE
```
Comment (Chelsea-Na) [inline scripts/data_guide/data_guide.qmd:456]: A SEG file (segmented data; .seg or .cbs) is a tab-delimited text file that lists loci and associated numeric values.

**Review by Chelsea-Na — CHANGES_REQUESTED** [review]
Wow, looks great.

Please see my comments

File: scripts/data_guide/data_guide.qmd [hunk 2]
```diff
@@ -0,0 +1,620 @@
...
+```{r}
+#| eval: true
+#| echo: false
+#| results: asis
+
+for (center in sort(processed_centers)) {
+  center = tolower(center)
+  a <- knitr::knit_child(glue::glue('genomic_profiles/{center}.qmd'), quiet=T)
+  cat(a, sep="\n")
+}
```
[the remark this answers is not in the document]
Comment (thomasyu888) [reply scripts/data_guide/data_guide.qmd:407]: Made this "Not Available"

File: scripts/data_guide/data_guide.qmd [hunk 3]
```diff
@@ -0,0 +1,620 @@
...
+
+ETHNICITY
+
+* Example Values: Non-Spanish/nonHispanic Spanish/Hispanic Unknown
+* Data Description: Indication of Spanish/Hispanic origin of the patient; this data element maps to the NAACCR v16, Element #190. Institutions not collecting Spanish/Hispanic origin have set this column to Unknown.
+
+ONCOTREE_CODE
+
+* Example Values: LUAD
+* Data Description: The primary cancer diagnosis "main type", based on the OncoTree ontology. The version of Oncotree ontology that was used for GENIE 13.0-public is oncotree_2021_11_02
```
Comment (Chelsea-Na) [inline scripts/data_guide/data_guide.qmd:483]: Update to use the release name parameter and to link out to the specific oncotree code

**Pull request merged**